Integrations/integration-PhishMe.yml

```diff
@@ -225,25 +292,47 @@ script:
         var md = "## PhishMe Hash Reputation for: "+str+"\n";
         var dbotScore = 0;
         var threatArray = [];
+        var threshold = params.fileThreshold;
+        var threshold_score = calcScore(threshold);
+        var indicatorFnd = false;
         if (threats.length !== 0) {
             ec.File = {
                         MD5: str,
                         Malicious: {Vendor: 'PhishMe', Description: 'Match found in PhishMe database'}
                         };
             for (var k = 0; k < threats.length; k++) {
-                for (var i = 0; i < threats[0].executableSet.length; i++){
-                    if (threats[0].executableSet[i]['md5Hex'] === str) {
-                        var severityLevel = threats[0].executableSet[i]['severityLevel'];
-                        break;
+                var severityLevel = 0;
+                for (var i = 0; i < threats[k].blockSet.length; i++){
+                    if (threats[k].blockSet[i]['impact']) {
+                        var threat_score = calcScore(threats[k].blockSet[i]['impact']);
+                        var adjusted_score = threshold_score <= threat_score ? 3 : threat_score;
+                        // if the queried indicator has a severity level, we'll take it
+                        if (threats[k].executableSet[i] && threats[k].executableSet[i]['md5Hex'] === str) {
```

Comment:
wh have here the same question about block lists with the indicator

Reply:
You mean the one about the indicator appearing in more than one block set?
The logic here has the same result - line 312 and 323 make sure that only the first indicator will affect the dbot score.